samples/rats-tls-client/client.c

```diff
@@ -226,7 +226,7 @@ int main(int argc, char **argv)
 	printf("    - Welcome to RATS-TLS sample client program\n");
 #endif
 
-	char *const short_options = "a:v:t:c:m:l:i:p:D:E:h";
+	char *const short_options = "a:v:t:c:ml:i:p:D:E:h";
```

Comment:
I think you need to read the man doc of getopt() function to know the purpose and meaning of the character ":". Additionally, the real bug here should be "D:". The option "D" don't require any options, so it should be "D" rather than "D:".